docs/using/alerting.md

```diff
@@ -1,44 +1,48 @@
 # Integrated Alerting
 
+*Integrated Alerting* lets you know when certain system events occur.
+
+!!! alert alert-warning "Warning"
+    Integrated alerting is a technical preview and is subject to change.
+
 ---
 
 [TOC]
 
 ---
 
-!!! alert alert-warning "Warning"
-    Integrated alerting is a technical preview and is subject to change.
-
-## Alerts
+## Definitions
 
-An alert has a name, a summary, a description, and a definition as a set of rules.
+- Alerts happen when their criteria (*alert rules*) are met; an *alert* is the result of an *alert rule* firing.
```

Comment:
> *alert* is the result of an *alert rule* firing

more like `an alert is the result of an alert rule expression evaluating to true`, because for an alert may not always be in a firing state. For e.g. if we use the MySQL down template to create an alert rule and enter 60s as the duration, if at some point in time the MySQL db goes down for let's say just 30s and alert is generated but in a pending state, it will only be in a firing state if it has existed for a duration longer than the one provided in our alert rule that is 60s in this example.

Reply:
>  Alerts happen

nitpick: I think "Alerts are generated" would be a better way to phrase this.